packages/atomic/src/components/common/facets/facet-common.tsx

```diff
@@ -386,8 +386,7 @@ export class FacetCommon {
   }
 
   private get numberOfSelectedValues() {
```

Comment:
I'm wondering if it it would be more appropriate to update the name of this getter to `numberOfActiveValues` now that it includes excluded values... You would have to change it in quite a few places, but it could easily be done using 1 big find/replace 🙂

Reply:
Good point. Would `numberOfSelectedValues` make more sense?

Reply:
> Would numberOfSelectedValues make more sense?

Hum isn't that the name already ?

Reply:
sorry, I meant `numberOfActiveValues`